### PR TITLE
[Content migration] - Asset Hub introduction, Register a Local Asset & Register a Foreign Asset

### PR DESCRIPTION
**Pull request by nhussein11**
This PR aims to bring content from the polkadot-ecosystem-docs-draft repository

Migrated content:

- _**Asset introduction**_ 
According to the spreadsheet, the location of this guide should be:
`Polkadot Protocol > System Parachains > Asset Hub > Introduction to Asset Hub`

- _**Register a Local Asset**_
According to the spreadsheet, the location of this guide should be:
`Develop > Parachain Developers > System Parachains > Register Your Native Asset on Asset Hub`

- _**Register a Foreign Asset**_
This one didn't exist on the spreadsheet, but I created a row there assuming that the location of this guide should be:
`Develop > Parachain Developers > System Parachains > Register Your Foreign Asset on Asset Hub`

### CODE REVIEW

File: develop/parachain-devs/system-parachains/register-a-foreign-asset-on-asset-hub.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+## Prerequisites
+
+The Asset Hub parachain is one of the system parachains on a relay chain, such as [Polkadot](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpolkadot.api.onfinality.io%2Fpublic-ws#/explorer){target=\_blank} or [Kusama](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkusama.api.onfinality.io%2Fpublic-ws#/explorer){target=\_blank}. To interact with these parachains, you can use the [Polkadot.js Apps](https://polkadot.js.org/apps/#/explorer){target=\_blank} interface for:
+
+- [Polkadot Asset Hub](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fasset-hub-polkadot-rpc.dwellir.com#/explorer){target=\_blank}
+- [Kusama Asset Hub](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fsys.ibp.network%2Fstatemine#/explorer){target=\_blank}
+
+For testing purposes, you can also interact with the Asset Hub instance on the following test networks:
+
+- [Rococo Asset Hub](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fasset-hub-rococo-rpc.dwellir.com#/explorer){target=\_blank}
```
Comment (eshaben) [inline develop/parachain-devs/system-parachains/register-a-foreign-asset-on-asset-hub.md:25]: Should we remove rococo now?

File: polkadot-protocol/system-parachains/asset-hub/asset-hub-introduction.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+### Foreign Assets
+
+Asset Hub considers assets originating from a different blockchain to be foreign assets. These assets can be native tokens from various parachains or other consensus systems like Ethereum. Once a foreign asset is added to Asset Hub, users can transfer this token from its original blockchain to Asset Hub and utilize it like any other asset.
+
+A significant difference lies in the method used for their identification. Unlike the Assets pallet, foreign assets use [XCM Multilocation](https://wiki.polkadot.network/docs/learn/xcm/fundamentals/multilocation-summary){target=\_blank} instead of integers to identify assets, making asset identification much more versatile.
+
+Foreign assets are implemented as an [instance of Assets pallet](https://github.com/paritytech/polkadot-sdk/blob/035211d707d0a74a2a768fd658160721f09d5b44/cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs#L408){target=\_blank}, but with a specialized configuration that enables support for XCM operations.
```
Comment (eshaben) [inline polkadot-protocol/system-parachains/asset-hub/asset-hub-introduction.md:47]: If we're removing rococo, we'll have to also update this link

**Review by eshaben — COMMENTED** [review]
Did a quick scan since we already reviewed this in the other repo. Everything looks good, just a minor question about removing rococo since it's being deprecated in a couple weeks

File: develop/parachain-devs/system-parachains/register-a-foreign-asset-on-asset-hub.md (hunk 1 above)
Comment (nhussein11) [reply develop/parachain-devs/system-parachains/register-a-foreign-asset-on-asset-hub.md:25]: Yeah, you're right

**Comment by eshaben** [comment]
@nhussein11 just some merge conflicts now

**Comment by nhussein11** [comment]
> @nhussein11 just some merge conflicts now

Done! now, this needs one approval to be merged @dawnkelly09 @eshaben 
Thanks!

**Review by CrackTheCode016 — APPROVED** [review]
LGTM

**Pull request merged**